public/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Library Project</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <h1>Books Library System</h1>
+    <form class="form" action="index.html">
+      <div class="">
+        <label for="title">Title</label>
+        <input type="text" name="title" />
```

Comment:
You might also want to add the `id` to each input to associate it with the label: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label.